backend/control_center/device_monitoring/migrations/0013_make_devicepingstats_hypertable.py

```
@@ -0,0 +1,35 @@
+def reverse_hypertable(apps, schema_editor):
+    """
+    Reverse migration is not supported for hypertables.
+    """
+    pass
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('device_monitoring', '0012_alter_devicepingstats_primary_key'),
+    ]
```

Ruff (0.14.1)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)